algoliasearch-common/src/main/java/com/algolia/search/APIClient.java

```diff
@@ -1362,6 +1380,61 @@ Task batchRules(
     return task.setAPIClient(this).setIndex(indexName);
   }
 
+  public TaskABTest AddABTest(ABTest abtest) throws AlgoliaException {
```

Comment:
methods in Java starts with lower case.
Seems all your methods are starting with capital case

Reply:
Indeed, will change it.

Reply:
Done. Waiting for Travis to run. Let me know if you see something else.